components/mediation/data-publishers/org.wso2.micro.integrator.prometheus.publisher/src/main/java/org/wso2/micro/integrator/prometheus/publisher/util/MetricFormatter.java

```diff
@@ -33,7 +32,6 @@
      * Return the type of metric
      *
      * @param type Metric Prometheus type
-     * @return Metric Type
      */
     private String typeString(Type type) {
```

Comment:
Javadoc does not align with the method signature

Reply:
Fixed in https://github.com/wso2/micro-integrator/pull/1551